packages/nextjs/app/types/data.ts

```diff
@@ -78,3 +80,10 @@ export interface SelectedVector {
   name: keyof ImpactVectors;
   weight: number;
 }
+
+export interface VectorList {
+  creator: Address;
+  title: string;
+  description: string;
+  vectors: SelectedVector[];
+}
```

Comment:
Thought I'd keep it simple to start, though maybe adding a `tags` field would be nice for users when searching for new lists